parser/sql/statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/ddl/MySQLCreateIndexStatement.java

```diff
@@ -17,11 +17,24 @@
 
 package org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.ddl;
 
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.ddl.table.AlgorithmTypeSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.ddl.table.LockTableSegment;
 import org.apache.shardingsphere.sql.parser.sql.common.statement.ddl.CreateIndexStatement;
 import org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.MySQLStatement;
 
+import java.util.Collection;
+import java.util.LinkedList;
+
 /**
  * MySQL create index statement.
  */
+@Setter
+@Getter
 public final class MySQLCreateIndexStatement extends CreateIndexStatement implements MySQLStatement {
+
+    private final Collection<AlgorithmTypeSegment> algorithmSegment = new LinkedList<>();
```

Comment:
When will there be multiple algorithm options?

Reply:
CREATE [UNIQUE | FULLTEXT | SPATIAL] INDEX index_name
    [index_type]
    ON tbl_name (key_part,...)
    [index_option]
    [algorithm_option | lock_option] ...

Sorry, I misunderstood the rules of MySQL syntax and thought it was a multi choice support. 
`[algorithm_option | lock_option] ...`